### PR TITLE
MVP-230 techdocs format update

### PR DESCRIPTION
**Pull request by TaylorFries**
## 🔰 Checklist

- [x] I have read and agree with the following checklist and am following the guidelines in our [Code of Conduct](/CODE_OF_CONDUCT.md) document.

> - I have performed a self-review of my code.
> - I have commented my code, particularly in hard-to-understand areas.
> - I have made corresponding changes to the documentation where required.
> - I have tested my changes to the best of my ability.
> - I have consulted with the team if introducing a new dependency.
> - My changes generate no new warnings.

### CODE REVIEW

File: techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Not Using a Proxy Pass
 
-> ### `Important`
-> This package is set up to use a proxy pass to the backend by default using `/api`.  
-> This means that all traffic to `<frontend url>/api` will be directed to `<backend_url>`.  
+!!! info "Important"
```
Comment (BradyMitch) [inline techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md:3]:
```suggestion
!!! important
```
Can we try this to see if it works?

File: techdocs/docs/getting-started/quick-start-guide.md [hunk 2]
```diff
@@ -24,9 +24,9 @@ root.render(
 );
 ```
 
-> ### `Note`
-> SSOProvider has optional props `backendURL`, `idpHint`, `onRefreshExpiry`, and `postLoginRedirectURL`.  
-> See TypeScript Type SSOProviderProps for more details on usage.
+!!! note "Note"
```
Comment (BradyMitch) [inline techdocs/docs/getting-started/quick-start-guide.md:27]:
```suggestion
!!! info "Note"
```
Can we try this to see which we like better?

File: techdocs/docs/index.md [hunk 3]
```diff
@@ -11,16 +11,16 @@ This npm package offers an integration solution for connecting React application
...
+    These packages complement, rather than replicate, Common Hosted Single Sign-On ([CSS]) or it's [example apps].  
+    They allow our own applications to integrate with the Common Hosted Single Sign-On ([CSS]) Service, providing us access to it's user management capabilities.  
+    They are developed and maintained by the [CITZ IMB Common Code] team, separate from the Common Hosted Single Sign-On ([CSS]) team.  
+    See why we made these packages at [Purpose and Benefits of This Package].
 
 <br />
 
-> ### `Tip`
-> If you don't have an SSO integration check out [SSO Integration Settings].
+!!! note "Tip"
```
Comment (BradyMitch) [inline techdocs/docs/index.md:22]:
```suggestion
!!! tip "Tip"
```
Again, try this

Comment (TaylorFries) [reply techdocs/docs/index.md]: tip_test

File: techdocs/docs/getting-started/quick-start-guide.md (hunk 2 above)
Comment (TaylorFries) [reply techdocs/docs/getting-started/quick-start-guide.md:27]: note preview:
note_test
info preview:
info_test

File: techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md (hunk 1 above)
Comment (TaylorFries) [reply techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md]: important preview: 
information_testpng

File: techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 # Not Using a Proxy Pass
 
-> ### `Important`
-> This package is set up to use a proxy pass to the backend by default using `/api`.  
-> This means that all traffic to `<frontend url>/api` will be directed to `<backend_url>`.  
+!!! important "Important"
```
Comment (BradyMitch) [inline techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md:3]: This just makes it look like the "note" so you  can change it back

Comment (TaylorFries) [reply techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md]: @BradyMitch what do you think of the "warning" blocks for the Information sections? 
Example:
warn_test

File: techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md [hunk 5]
```diff
@@ -1,8 +1,8 @@
 # Not Using a Proxy Pass
 
-> ### `Important`
-> This package is set up to use a proxy pass to the backend by default using `/api`.  
-> This means that all traffic to `<frontend url>/api` will be directed to `<backend_url>`.  
+!!! warning "Important"
```
Comment (BradyMitch) [inline techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md:3]: Are we using warning or info for the "Important" notes? I think either works

File: techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md (hunk 4 above)
Comment (BradyMitch) [reply techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md]: I think both info and warning look good. I think maybe warning stands out more.

File: techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md (hunk 5 above)
Comment (BradyMitch) [reply techdocs/docs/getting-started/proxy-pass/not-using-a-proxy-pass.md:3]: As noted in above conversation

File: techdocs/docs/getting-started/quick-start-guide.md [hunk 6]
```diff
@@ -58,22 +58,22 @@ Conditionally render a Login or Logout button:
...
-> This package is set up to use a proxy pass to the backend by default using `/api`.  
-> This means that all traffic to `<frontend url>/api` will be directed to `<backend_url>`.  
-> If your application is not using a proxy pass, or is using a different proxy follow the steps below.
+!!! warning "Important"
+    This package is set up to use a proxy pass to the backend by default using `/api`.  
+    This means that all traffic to `<frontend url>/api` will be directed to `<backend_url>`.  
+    If your application is not using a proxy pass, or is using a different proxy follow the steps below.
 
-To set up a proxy using `vite` and `nginx` see [Setting Up a Proxy Pass](https://github.com/bcgov/citz-imb-sso-react/techdocs/docs/Getting-Started/Proxy-Pass/Setting-Up-a-Proxy-Pass.md).
+To set up a proxy using `vite` and `nginx` see [Setting Up a Proxy Pass](/getting-started/proxy-pass/setting-up-a-proxy-pass.md).
```
Comment (BradyMitch) [inline techdocs/docs/getting-started/quick-start-guide.md:76]:
```suggestion
To set up a proxy using `vite` and `nginx` see [Setting Up a Proxy Pass](/docs/default/component/citz-imb-sso-react-npm-package/getting-started/proxy-pass/setting-up-a-proxy-pass.md).
```

Comment (TaylorFries) [reply techdocs/docs/getting-started/quick-start-guide.md]: this didnt work either

**Review by BradyMitch — APPROVED** [review]

**Pull request merged**